StyleGuide.md

```diff
@@ -535,52 +540,66 @@ Do not use underlined text in CockroachDB docs. If it seems beneficial to emphas
+#### Link capitalization
 
 Link capitalization should match our [capitalization rules](#capitalization-rules) for page titles and headers:
 
 - **Use title case** when referring to the linked doc by its page title (e.g., "See __Best Practices__ for more information").
 - **Use sentence case** when referring to the linked doc by one of its headers (e.g., "See __Clock synchronization__ for further guidance").
 - **Use sentence case** - when referring to a linked doc without explicitly citing a page title or header (e.g., "[…] follow the __identifier rules__ when creating […]").
 
-Links are marked with inline text surrounded by square brackets followed by the link address in parentheses. If you are including a relative (i.e., internal) link:
+#### Links to CockroachDB docs pages in the same folder
 
-- To link to another page in the docs, use just the name of the file.
+To link to a page within the same folder (e.g., a page in `v23.1` to another page in `v23.1`), use the [Jekyll link syntax](https://jekyllrb.com/docs/liquid/tags/#links).
 
-    **Example:** `[here](name-of-article.html)`
+**Example:** `[Foreign Key Constraint]({% link v23.1/foreign-key.md %})`
 
-- To link to a specific heading on another page, use the name of the file plus the heading.
+To include a subsection, place it outside of the Liquid tag.
 
-    **Example:** `[xyz](name-of-article.html#heading-on-page)`
+**Example:** `[Rules for creating foreign keys]({% link v23.1/foreign-key.md %}#rules-for-creating-foreign-keys)`
 
-- To link to a specific heading on the current page, use just the heading.
+This also applies to files within a subfolder of the same folder (e.g., a link from `v23.1/abc.md` to `v23.1/architecture/xyz.md` or from `v23.1/architecture/xyz.md` to `v23.1/abc.md`).
 
-    **Example:** `[xyz](#heading-on-page)`
+**Example:** `[Multi-active availability]({% link v23.1/architecture/glossary.md %}#multi-active-availability)`
 
-- To link to a specific location on a page that is not a heading (e.g., a specific command-line flag in a table), add a manual anchor and use the `name` parameter:
+#### Links to CockroachDB docs pages outside of the current folder
 
-    **Example:**
+To link to a page outside of the current folder (e.g., a link from `v23.1` to `cockroachcloud`), use the fully qualified production URL:
 
-    ```
-    # Anchor:
-    <a name="flags-max-offset"></a>`--max-offset`
-    ```
+**Example:** `[Quickstart with CockroachDB](https://www.cockroachlabs.com/docs/cockroachcloud/quickstart)`
 
-    ```
-    # Link:
-    [--max-offset](#flags-max-offset)
-    ```
+#### Links to a specific location on a page that is not a heading
+
+To link to a specific location on a page that is not a heading (e.g., a specific command-line flag in a table), add a manual anchor and use the `name` parameter:
+
+**Example:**
+
+```
+# Anchor:
+<a name="flags-max-offset"></a>`--max-offset`
```

Comment:
Is it better to use `id=` instead of `name=` since `name` doesn't need to be unique on a page?